components/Row.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from "react";
+import Box from "@mui/material/Box";
+import Collapse from "@mui/material/Collapse";
+import IconButton from "@mui/material/IconButton";
+import Table from "@mui/material/Table";
+import TableCell from "@mui/material/TableCell";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
+import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
+import { makeStyles } from "@mui/styles";
+
+import { Colours } from "./Colours";
+
+import Tabs from "./Tabs";
+import { DependencyData, semVerToString } from "./dataProcessing";
+
+import styles from "../components/row.module.css";
+import { Circle } from "@mui/icons-material";
+import { Typography } from "@mui/material";
```

Comment:
might be beneficial to re-organise imports based on where they are coming from, i.e. mui, our components, etc.